src/components/asthma/components/AsthmaBarChart/AsthmaBarChart.tsx

```diff
@@ -0,0 +1,152 @@
+import React, { useState } from 'react';
+import { checkDailyDataAvailability, DailyDataProvider, DailyDataQueryResult, queryDailyData } from '../../../../helpers/query-daily-data';
+import { LoadingIndicator } from '../../../presentational';
+import { add, format, startOfToday } from 'date-fns';
+import getDayKey from '../../../../helpers/get-day-key';
+import { AxisDomain } from 'recharts/types/util/types';
+import { Bar, BarChart, CartesianGrid, Cell, LabelList, ResponsiveContainer, XAxis, YAxis } from 'recharts';
+import { useInitializeView } from '../../../../helpers/Initialization';
+import './AsthmaBarChart.css';
+
+export interface AsthmaBarChartProps {
```

Comment:
Does this need to be coupled with Asthma? Just a bar chart
Also, need stories?

Reply:
Functionally it is not much different than the DailyDataChart set to start 7 days ago w/ no date range nav but it is a bit more refined and obviously supports highlighting/formatting...(and there is no reason for us to only support a single style of bar chart...)

Reply:
I've moved it out to the core components as `RecentDailyDataBarChart`.